lr-cli/cortx_setup/commands/storage/config.py

```diff
@@ -107,6 +107,24 @@ class StorageEnclosureConfig(Command):
             'default': None,
             'optional': True,
             'help': 'Port of the controller to connect to'
+        },
+        'cvg': {
+            'type': int,
```

Comment:
Can we fix this as unsigned integer with value always greater than 0

Reply:
I am not Python expert but as far as know there' s no built-in data type "unsigned int" in Python, please let know if there is any other way, but I am using -1 as a default value to initialize cvg, this is required to check if user has skipped the parameter in the command.